db/migrate/20160427110629_create_service_interactions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateServiceInteractions < ActiveRecord::Migration
+  def change
+    create_table :service_interactions do |t|
+      t.integer :service_identifier
+      t.string :service_label
+      t.integer :ineraction_identifier
+      t.string :interaction_label
```

Comment:
It would probably be more useful if instead of storing the service and interaction identifiers and labels we stored the database ids of the Service and Interaction instances that have those identifiers.  This would let us hook up Service and Interaction via rails associations.

It would make importing more complex as we'd have to make sure we'd imported all the services and interactions first before importing the allowed combinations.  It does however give us scope for errors where try to import a combination of service 123 and interaction 456 but one (or both) of them don't exist yet.  This'll help us expose holes in the data (which *should* be none as the service, interaction, and valid combinations lists all come from the same source, but you never know).